saturn-rapids/environment.yml

```diff
@@ -24,13 +23,13 @@ dependencies:
 - pyarrow
 - python
 - python-graphviz
-- rapids-blazing=21.06.01=cuda11.2_py37_gecaf371_431
+- rapids-blazing=21.10.00=cuda11.2_py38_ge66f011_114
```

Comment:
I know this is what we're currently doing, And I don't have really strong opinions - but why would we not just use the rapids metapackage?

Reply:
yeah I'm not sure I wasn't really thinking critically about it